src/object/get.js

```diff
@@ -4,9 +4,9 @@ define(['../lang/isPrimitive'], function (isPrimitive) {
      * get "nested" object property
      */
     function get(obj, prop){
+        if (!obj) return;
```

Comment:
This would also be true of `obj` being `false`, `0`. And essentially everything that evaluates to `false`. I guess that is what we want, I can't really think of anything else, just making sure.